packages/alfa-style/src/property/helpers/mask-layers.ts

```diff
@@ -0,0 +1,38 @@
+import { List, type Value } from "@siteimprove/alfa-css";
+
+import type { Style } from "../../style.js";
+
+/**
+ * {@link https://drafts.fxtf.org/css-masking/#layering}.
+ *
+ * @remarks
+ * The computed value depends on the number of layers.
+ * A layer is created for each of the comma separated values for `mask-image`.
+ *
+ * If there are more values than layers, the excess values are discarded.
+ * Otherwise, the values must be repeated until the number of values matches the number of layers.
+ */
+export function matchLayers<V extends Value>(
+  value: List<V>,
+  style: Style,
+): List<V> {
+  const numberOfLayers = Math.max(
+    style.computed("mask-image").value.values.length,
```

Comment:
**Suggestion:** This feels like we should add a `List#size` and use it instead of `List#values.length`.